autogluon/scheduler/fifo.py

```diff
@@ -257,6 +257,10 @@ def _run_reporter(self, task, task_job, reporter, searcher):
         last_result = None
         while not task_job.done():
             reported_result = reporter.fetch()
+            if 'traceback' in reported_result:
```

Comment:
I can confirm that the reaction to an exception being reported (the signal is whether 'traceback' is present) is OK. What happens is that the _run_reporter while loop is left. The current reported_result is not used as a datapoint for the model (this is correct -- it does not have any metric value). What *is* used is the last recently fetched reported_result which did not fail. If the process reports an exception before ever obtaining a metric value, then nothing is reported to the model.

For most searchers using FIFO scheduler, this should be a good reaction. We want to make sure that a config whose evaluation has failed, is not attempted again. At least for RandomSearcher, this currently works, because a config is never attempted twice in the first place.